.travis.yml

```diff
@@ -1,29 +1,19 @@
 language: objective-c
-osx_image: xcode7.3
-xcode_sdk: iphonesimulator9.3
+osx_image: xcode8
+xcode_sdk: iphonesimulator10.0
 
 env:
   global:
   - WORKSPACE=SparkSDK.xcworkspace
   - SCHEME=SparkSDK
   matrix:
-    - DESTINATION="OS=9.0,name=iPhone 5"       CONFIGURATION="Debug"
-    - DESTINATION="OS=9.0,name=iPhone 5"       CONFIGURATION="ReleaseTest"
-    - DESTINATION="OS=9.1,name=iPhone 5s"      CONFIGURATION="Debug"
-    - DESTINATION="OS=9.1,name=iPhone 5s"      CONFIGURATION="ReleaseTest"
-    - DESTINATION="OS=9.0,name=iPhone 6"       CONFIGURATION="Debug"
-    - DESTINATION="OS=9.0,name=iPhone 6"       CONFIGURATION="ReleaseTest"
-    - DESTINATION="OS=9.1,name=iPhone 6 Plus"  CONFIGURATION="Debug"
-    - DESTINATION="OS=9.1,name=iPhone 6 Plus"  CONFIGURATION="ReleaseTest"
-    - DESTINATION="OS=9.2,name=iPhone 6s"      CONFIGURATION="Debug"
-    - DESTINATION="OS=9.2,name=iPhone 6s"      CONFIGURATION="ReleaseTest"
-    - DESTINATION="OS=9.3,name=iPhone 6s Plus" CONFIGURATION="Debug"
-    - DESTINATION="OS=9.3,name=iPhone 6s Plus" CONFIGURATION="ReleaseTest"
+    - DESTINATION="OS=10.0,name=iPhone 6s"      CONFIGURATION="Debug"
+    - DESTINATION="OS=10.0,name=iPhone 6s"      CONFIGURATION="ReleaseTest"
 
 script:
   - set -o pipefail
   - xcodebuild -workspace "$WORKSPACE" -scheme "$SCHEME" -destination "$DESTINATION" -configuration "$CONFIGURATION" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO clientSecret="$CLIENTSECRET" test | xcpretty -c
 
 branches:
   only:
-    - master
```

Comment:
We should not change the active branch. The CI will run against every pull request.